src/common/interfaces.ts

```diff
@@ -71,62 +48,26 @@ export interface IWorkerInputBase {
   gpkgEstimatedSize?: number;
   targetFormat?: TileOutputFormat;
   outputFormatStrategy?: TileFormatStrategy;
-}
-
-/**
- * @deprecated GetMap API - will be deprecated on future
- */
-export interface IWorkerInput extends IWorkerInputBase {
-  targetResolution: number;
-  fileName: string;
-  callbacks: ICallbackTarget[];
-  sanitizedBbox: BBox;
-  zoomLevel: number;
-}
-
-export interface IWorkerExportInput extends IWorkerInputBase {
   callbacks?: ICallbackTargetExport[];
   roi: FeatureCollection;
   fileNamesTemplates: ILinkDefinition;
   description?: string;
+  traceContext: ITraceParentContext;
```

Comment:
Consider extends the `ITraceParentContext` for each interface that should extend it instead of repeat that field definition it on each interface.